dts/bindings/flash_controller/st,stm32l0-flash-controller.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020, Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 L0 flash controller
```

Comment:
Since STM32L0 flash driver is not supported, I'd prefer to remove the flash driver node in stm32l0.dtsi

Reply:
Why?   The device tree should represent the hardware.  Part of the reason I'm making these clean ups is I want to require that 'soc-nv-flash' is a child of an actual flash controller.  So removing this for L0 would violate that.

Reply:
Ok my point is I'd like to see dts to reflect actual support in zephyr:
If a node is present, it is supported.
Since L0 flash driver is not implemented, having it in dts makes this confusing.
But if this is not possible given the 'soc-nv-flash' availability, then a comment should be (/have already been) added to make it clear.

Reply:
Where should I add the comment?

Reply:
Comment added.